tools/build_virtual_environment/dynamic_ve/Dockerfile.canary_to_vip

```diff
@@ -0,0 +1,11 @@
+FROM atsigncompany/dynamicvirtualenv:canary

```

Comment:
Do we expect to have canaries for this, or has this just been copied over?

Reply:
copied over, I can remove it